src/main/java/org/sonarlint/intellij/editor/SonarExternalAnnotator.java

```diff
@@ -196,8 +181,16 @@ private static HighlightSeverity getSeverity(@Nullable String severity) {
     return SonarLintSeverity.byName(severity).highlightSeverity();
   }
 
-  private static TextRange createTextRange(RangeMarker rangeMarker) {
-    return new TextRange(rangeMarker.getStartOffset(), rangeMarker.getEndOffset());
+  @CheckForNull
+  private static TextRange toValidTextRange(@Nullable RangeMarker rangeMarker) {
```

Comment:
Is it used somewhere?

Reply:
Good catch! Forgot to remove it after a move